cdk_emqx_cluster/cdk_emqx_cluster_stack.py

```diff
@@ -449,6 +449,14 @@ def setup_monitoring(self, targets):
                                             # fail
                                             'PGDATA': '/var/lib/postgresql/pgdata'
                                         },
+                                        # uncomment to reset the WAL; it may resolve a
```

Comment:
Fail & retry feels very expensive in terms of time and cost. 
Do we know any safe method to turn off the postgres container when we destroy the cluster?

If no, pls report an issue for this so that we could track it.

Reply:
Yes, it's very time consuming every time this happens... The first time in particular, before I knew that this was the problem. :see_no_evil: 

Unfortunately, I don't know how to make it safe yet.  I tried to increase the stop timeout in #48 , but the error still occurred to me recently.

I created this issue to track it: #54 .